### ISSUE TITLE
Name error raised while converting

### ISSUE THREAD

**Issue by PPTA22** [snoopyjc/xls2xlsx#4]
* xls2xlsx version: 0.1.3
* Python version: 3.7.9
* Operating System: Win 10 Pro, 1909

### Description

Name error raised while converting to xlsx, I am not sure is the excel problem or the lib problem

I tried to use the `Save as` to output as different xls file in the Excel 2010, but the same error still occur
The excel file is locked to prevent unwanted change, so I can only read this file, I am not sure is this the reason cause the error.

If you need the xls file, I can send it to you

### What I Did

```
>>> os.listdir()
['OOCL.XLS']
>>> from xls2xlsx import XLS2XLSX
>>> x2x = XLS2XLSX(os.listdir()[0])
>>> x2x.to_xlsx()
Traceback (most recent call last):
  File "<input>", line 1, in <module>
  File "C:\Users\####\Anaconda3\envs\ocean2\lib\site-packages\xls2xlsx\xls2xlsx.py", line 240, in to_xlsx
    alignment = copy.deepcopy(alignment)
NameError: name 'copy' is not defined
```

**Comment by PPTA22** [comment]
[OOCL.zip](https://github.com/snoopyjc/xls2xlsx/files/5478805/OOCL.zip)

**Comment by snoopyjc** [comment]
Thank you - looks like I need to add a 

    import copy

and a new test case!  I will fix it soon. 

-joe

**Comment by PPTA22** [comment]
Thanks 👍

**Comment by snoopyjc** [comment]
Fixed in 0.1.4
